File: src/main/java/xin/xiuyuan/wecomchan/model/MsgResult.java
package xin.xiuyuan.wecomchan.model;

import lombok.Data;
import lombok.experimental.Accessors;

/**
 * @author xinbj
 * @date 2021-11-11 9:23
 */
@Data
@Accessors(chain = true)
public class MsgResult {

    /**
     * 返回码
     */
    private Integer errcode;

    /**
     * 返回消息
     */
    private String errmsg;

    /**
     * 消息ID
     */
    private String msgid;


}

File: models/msg_result.go
package models

type MsgResult struct {
	//错误码
	ErrCode int `json:"errcode"`
	//错误信息
	ErrMsg string `json:"errmsg"`
	//消息ID
	MsgId string `json:"msgid"`
	//授权码
	AccessToken string `json:"access_token"`
	//有效期 单位秒
	ExpiresIn int `json:"expires_in"`
}
